src/components/layout/AppContent/AppContent.tsx

```diff
@@ -1,10 +1,27 @@
 import './AppContent.css'
+import { AppCategoryOptions } from "../../common/AppCategoryOptions/AppCategoryOptions.tsx";
+import {
+    CategoryOptionDataContainer
+} from "../../features/CategoryOptionDataContainer/CategoryOptionDataContainer.tsx";
 
-export const AppContent = () => {
+interface IAppContentProps {
+    isFullScreenContent: boolean
+}
 
-    return (
-        <div className="content-container">
+export const AppContent = ({isFullScreenContent}: IAppContentProps) => {
+    const windowPathName = window.location.pathname
 
+
+    return (
+        <div className={`content-container ${isFullScreenContent ? "notFullScreenWidth" : "fullScreenWidth"}`}>
+            <div>
+                {windowPathName !== '/' &&
```

Comment:
Better use https://github.com/JedWatson/classnames for refer to class names from CSS